### DISCUSSION TITLE
Finger print sensor for cars

### DISCUSSION THREAD

**Discussion by anirudh-IITISM** [scaleracademy/open-source-projects#77] [Ideas]
Finger sensors fitted on car door handles.
Recognises person from the finger print and not only unlocks the car but does additional things like setting a preferred AC temp,seat height , steering wheel height and more connected features.
Can do away with the use and need of a key altogether

**Comment by ShardulDS** [comment]
Considering that we will have to make a key for failsafe measures, we can add a tracking chip to it so that the car can detect when its driver is nearby and turn on the fingerprint scanner and make it faster to turn on the car. It can also turn on the additional features like ac, seat heater and steering wheel height.

**Comment by Akshatpattiwar512** [comment]
The Identisafe-2017 (GOD) Biometric Anti-Theft Device is a fingerprint Bluetooth CAN Bus engine immobilizer that protects your car without the necessity for wire cutting or the addition of after-market key fobs.
After integrating Identisafe-2017 (GOD) into the vehicle's CAN bus data network, the vehicle cannot be operated without an authorized fingerprint.
Each automobile has a memory of ten drivers.
To avoid highway accidents, use speed control.
This capability allows each motorist to establish a speed restriction for himself by registering his fingerprint as an authorized driver on Identisafe-2017 (GOD).

**Comment by johnwill0** [comment]
You're right anirudh. These things work mostly in high cc cars. Everyone can not afford these luxuries. So in case if you face any car accident injury in Atlanta, Georgia I found a very trustworthy [lawyer](https://www.reedinjurylaw.com/) over there. Actually last month I face this issue and he helped me a lot to win my case.

**Comment by gthakurx** [comment]
As per my understanding 
1. You need to integrate the sensor first on the Car SOC . 
2. Car SOC will populate the data based on different users
